### DISCUSSION TITLE
Google地圖增加新功能 告訴你哪裡正在失火

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#81] [General]
Google Maps上新增了野火圖層（Wildfire Layer），這可用來觀看全球的野火位置與資訊。在啟用野火圖層之後，就能一次瀏覽全球的野火資訊，當使用者點擊每一個野火，就能看到當地政府所提供的資源，諸如緊急設立的網站、求助電話或是疏散資訊等，或者是關於野火的遏制資訊、範圍，以及更新時間等。
樹冠（Tree Canopy）工具
樹冠實驗室，它是Google環境洞察探索(EIE）平臺上的專案之一，EIE是藉由資料及建模來協助各大城市測量與分析碳排放來源，以制定減少碳排放的策略。樹冠指的則是樹木頂端的枝葉層，用來評估城市的樹木覆蓋率。其樹冠工具是利用飛機在天空中所拍攝的照片，並輔以各種AI能力，找出一個城市中的熱島，即氣溫上升最快的地區。Google表示，這些地區經常是低收入社區，不僅空氣品質不佳，還經常缺水，各地政府將可藉由樹冠工具辨識應在哪些區域種植樹木以增加遮蔭範圍、減少熱氣與降低不良影響。
https://tw.news.yahoo.com/google%E5%9C%B0%E5%9C%96%E5%A2%9E%E5%8A%A0%E6%96%B0%E5%8A%9F%E8%83%BD-%E5%91%8A%E8%A8%B4%E4%BD%A0%E5%93%AA%E8%A3%A1%E6%AD%A3%E5%9C%A8%E5%A4%B1%E7%81%AB-111139588.html

**Comment by EricAgumonJou** [comment]
感謝分享~
看來加州野火有得救了, 甚至可以提早防範大火發生. :)

**Comment by enoch-lightning** [comment]
能掌握即時野火的資訊，已經很不錯。下一步應是要結合作為樹冠分析的飛機，可能是無人機，即時生成人造雨，以減弱，甚至消除火災。

**Comment by chupochun** [comment]
感謝分享 相信不久的以後一定會有更多 類似的救災機器出現

**Comment by pekoranoinu** [comment]
google map 的功能真是越來越多樣化了。

**Comment by ThomasMeng0515** [comment]
google map確實功能越來越多樣化，現在還會標記出人潮壅擠的地區，方便出去遊玩的時候即時了解旅遊景點的人數多寡。
